tests/Makefile

```diff
@@ -15,6 +15,10 @@ text-test-data:
 	hg clone https://bitbucket.org/bos/text-test-data
 	$(MAKE) -C text-test-data
 
+text-test-data-git:
```

Comment:
There's a difference to the `text-test-data` target though: the `text-test-data` target actually creates the filename it's named as, whereas `text-test-data-git` doesn't result in a folder `text-test-data-git` being created.

Reply:
@hvr: true, thanks; i've updated the commit